site/docs/master/httpproxy.md

```diff
@@ -252,6 +252,54 @@ The TLS **Minimum Protocol Version** a vhost should negotiate can be specified b
 - 1.2
 - 1.1 (Default)
 
+##### Fallback Certificate
+
+Contour provides virtual host based routing, so that any TLS request is routed to the appropriate service based on the hostname specified in the request's HOST header.
```

Comment:
"based on both the server name requested by the TLS client and the HOST header in the HTTP request".